com.unity.multiplayer.mlapi/Tests/Editor/NetworkObjectTests.cs

```diff
@@ -6,6 +6,30 @@ namespace MLAPI.EditorTests
 {
     public class NetworkObjectTests
     {
+        [Test]
+        public void NetworkManagerOverrideTest()
+        {
+            // Create "bait"
+            var singletonNetworkManager = new GameObject(nameof(NetworkManager)).AddComponent<NetworkManager>();
+            singletonNetworkManager.SetSingleton();
+
+            // Create override
+            var networkManager = new GameObject(nameof(NetworkManager)).AddComponent<NetworkManager>();
+
+            // NetworkObject
+            var gameObject = new GameObject(nameof(NetworkManagerOverrideTest));
+            var networkObject = gameObject.AddComponent<NetworkObject>();
+
+            // Set override
+            networkObject.NetworkManagerTestOverride = networkManager;
+
+            Debug.Assert(networkObject.NetworkManager == networkManager);
```

Comment:
Especially if you agree with my above comment, a good test would be to create a list of objects that belong to different NM's and then call NetworkHide / NetworkShow and make sure it works.

Reply:
It's not meant to work :)

If you call the **static** versions of NetworkShow and NetworkHide on objects with different NetworkManagers it should throw an exception.